src/test/groovy/com/intuit/graphql/orchestrator/datafetcher/ServiceDataFetcherSpec.groovy

```diff
@@ -1,28 +1,30 @@
-package com.intuit.graphql.orchestrator.datafetcher;
+package com.intuit.graphql.orchestrator.datafetcher
 
-//public class ServiceDataFetcherTest {
+//class ServiceDataFetcherTest extends Specification {
```

Comment:
👍  we need to make this test working in the near future.  Created a new Issue in github: https://github.com/intuit/graphql-orchestrator-java/issues/109

Reply:
Created: https://jira.intuit.com/browse/DOS-1571